tests/dashbio_demos/app_alignment_viewer.py

```diff
@@ -92,204 +93,167 @@ def description():
-                                    html.P(
-                                        """
-                                        Note that the AlignmentChart only returns a chart of
-                                        the sequence, while AlignmentViewer has integrated
-                                        controls for colorscale, heatmaps, and subplots allowing
-                                        the user to interactively control their sequences.
-                                        """
-                                    ),
-                                    html.P(
-                                        """
-                                        Read more about the component here:
-                                        https://github.com/plotly/react-alignment-viewer
-                                        """
-                                    ),
-                                ], className='alignment-subcard')
-                            ],
+                            label='About',
+                            value='alignment-tab-about',
+                            children=html.Div(className='alignment-tab', children=[
+                                html.H4(
+                                    "What is Alignment Viewer?"
+                                ),
+                                html.P(
+                                    """
+                                    The Alignment Viewer (MSA) component is used to align
+                                    multiple genomic or proteomic sequences from a FASTA or
+                                    Clustal file. Among its extensive set of features,
+                                    the multiple sequence alignment viewer can display
+                                    multiple subplots showing gap and conservation info,
+                                    alongside industry standard colorscale support and
+                                    consensus sequence. No matter what size your alignment
+                                    is, Alignment Viewer is able to display your genes or
+                                    proteins snappily thanks to the underlying WebGL
+                                    architecture powering the component. You can quickly
+                                    scroll through your long sequence with a slider or a
+                                    heatmap overview.
+                                    """
+                                ),
+                                html.P(
+                                    """
+                                    Note that the AlignmentChart only returns a chart of
+                                    the sequence, while AlignmentViewer has integrated
+                                    controls for colorscale, heatmaps, and subplots allowing
+                                    the user to interactively control their sequences.
```

Comment:
```suggestion
                                    you to control interactively your sequences.
```
Because the previous paragraph was written for "you" and not for "the user."